lib/worker_tools/xlsx_output.rb

```diff
@@ -103,25 +92,41 @@ def xlsx_style_columns(spreadsheet, styles, headers)
       true
     end
 
-    def xlsx_write_sheet(workbook, sheet_content, index)
+    def xlsx_output_tmp_file
```

Comment:
I associate tmp with temporary. I would abbreviate template with tpl.

Reply:
it is not what it seems, a bunch of renames and reordering was too much to ask for the diff 😂 

`xlsx_write_sheet` is still there with the name `xlsx_output_write_sheet`

`xlsx_output_tmp_file` is the file destination, an actual temporary file